content/2017.12.kinto-at-mozilla.rst

```diff
@@ -0,0 +1,119 @@
+During the last company «all hands», we've heard many mentions of *Kinto*, mostly from people that we hadn't met before or that wasn't aware we were the authors. Part of this fame may probably be due to the success of Firefox Notes, and the recent addition of the long awaited synchronization features.
+
+It's amazing how we've accomplished with this project in very heterogenous contexts, and sadly we haven't really taken the time to celebrate. This post intends to present the different use-cases we are aware of at Mozilla, with some architecture and metrics details.
+
+
+Firefox remote settings
+=======================
+
+The historical one!
+
+Firefox addons and security engineers edit remote settings like blocklist or certificates revocation lists via the Kinto Admin UI. Reviewing and sign-off features are powered by `kinto-signer`. The several hundreds millions of Firefox browsers regularly poll for changes using the `kinto-changes` endpoint. Two instances are deployed with the same database, a public one read-only behind a CDN, and a private one only accessible via a secured VPN.
```

Comment:
I think we should remove several: The hundreds' millions Firefox browsers

Reply:
We can say that during the #yallhands we decided to start implementing a push notification for changes rather than daily polling.

Reply:
Well you said it below. 💃